concreate/generator.py

```diff
@@ -20,7 +20,7 @@ class Generator(object):
 
     def __init__(self, descriptor_path, target, overrides):
 
-        self.descriptor = Descriptor(descriptor_path, 'image').process()
+        self.descriptor = Image(tools.load_descriptor(descriptor_path))
```

Comment:
I would say `self.descriptor` is a misleading name for an field now.

Reply:
Do you have any suggestion? Its still a descriptor - only difference is that it is divided into containers not plain dictionaries.